appveyor.yml

```diff
@@ -14,3 +14,9 @@ build:
 configuration:
 - Debug
 - Release
+test:
+  assemblies:
```

Comment:
Response from Nunit authors: https://github.com/nunit/nunit/issues/2082
This is a different behavior between nunit 2 runner and nunit 3 runner.
With nunit 3 runner, if an assembly DOES reference nunit.framework (as HttpMock.dll does), the assembly should HAVE a NonTestAssemblyAttribute (HttpMock doesn't have this attribute), and in the command-line with `--skipnontestassemblies`, then the runner will pass it. Otherwise, it is treated as INVALID.

So for now the only way I can do is to use the appveyor config file to skip the HttpMock.dll.